components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -1166,21 +1166,44 @@ func (p *pg) GetStagedOrAppliedRule(ctx context.Context, projectID string, ruleI
 	return &rule, nil
 }
 
+func (p *pg) MapAllAppliedRulesToProjects(ctx context.Context) (map[string][]*v2.Rule, error) {
```

Comment:
[nit] This sounds like it was _doing_ something; but it's only data retrieval. I'd be a little less confused if it was named `AppliedRulesByProjectIDs` or something like that 🤔

Reply:
How about `FetchProjectRulesMap`?

Reply:
`FetchProjectAppliedRulesMap`